db/install.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<XMLDB PATH="blocks/course_template" VERSION="20120508" COMMENT="XMLDB file for Moodle course template block"
+<XMLDB PATH="blocks/course_template/db" VERSION="20120508" COMMENT="XMLDB file for Moodle course template block"
```

Comment:
this makes me nervous - it looks like you have edited install.xml by hand - ideally you should open this in the xmldb editor, to trigger a change I usually edit a comment, save, and then revert the change. It can also be good to use the built-in save functionality rather than trying to copy/paste - so you need to temporarily make the file writable by your local web server. Save the file using xmldb, then revert the file permission changes and then commit.